jina/proto/jina.proto

```diff
@@ -106,6 +106,9 @@ message DocumentProto {
 
         // a uri of the document could be: a local file path, a remote url starts with http or https or data URI scheme
         string uri = 9;
+
+        // Related information to be used when a Document represents a graph with its chunks as nodes
+        GraphProto graph_info = 27;
```

Comment:
how about just `graph`

Reply:
yes, may make sense.

What about the `traversal-path` I comment in the `Description`, any opinion?